src/ocean/text/util/StringC.d

```diff
@@ -89,7 +89,9 @@ class StringC
     {
         if (str.length && !!str[$ - 1])
         {
+            enableStomping(str);
             str ~= StringC.Term;
```

Comment:
> I missed the point of shrinking length. But, is then this PR even necessary? I don't think so?

Depends on where the buffer came from. If it was from your app, PR is not necessary. If your app called some other ocean utility which called this, that ocean utility may need to be adjusted instead.